matplotcheck/tests/test_polygons.py

```diff
@@ -2,82 +2,124 @@
 import pytest
 import matplotlib.pyplot as plt
 import geopandas as gpd
+from shapely.geometry import Polygon
 from matplotcheck.vector import VectorTester
+import matplotlib
+
+matplotlib.use("Agg")
 
 
 @pytest.fixture
-def poly_geo_plot(basic_polygon_gdf):
-    """Create a polygon vector tester object."""
+def multi_polygon_gdf(basic_polygon):
+    """
+    A GeoDataFrame containing the basic polygon geometry.
+    Returns
+    -------
+    GeoDataFrame containing the basic_polygon polygon.
+    """
+    poly_a = Polygon([(3, 5), (2, 3.25), (5.25, 6), (2.25, 2), (2, 2)])
+    gdf = gpd.GeoDataFrame(
+        [1, 2], geometry=[poly_a, basic_polygon], crs="epsg:4326",
+    )
+    multi_gdf = gpd.GeoDataFrame(
+        geometry=gpd.GeoSeries(gdf.unary_union), crs="epsg:4326"
+    )
+    return multi_gdf
+
+
+@pytest.fixture
+def poly_line_plot(two_line_gdf):
+    """Create a line vector tester object."""
     _, ax = plt.subplots()
 
-    basic_polygon_gdf.plot(ax=ax)
-    ax.set_title("My Plot Title", fontsize=30)
-    ax.set_xlabel("x label")
-    ax.set_ylabel("y label")
+    two_line_gdf.plot(ax=ax)
 
     axis = plt.gca()
 
     return VectorTester(axis)
 
 
 @pytest.fixture
-def point_geo_plot(pd_gdf):
-    """Create a point plot for testing"""
+def poly_multiline_plot(multi_line_gdf):
```

Comment:
just curious. this says poly multiline? is it a multiline object or a polygon? please clarify

Reply:
Sorry, this is a bad name. I'm going to rename it `multiline_geo_plot` to more closely match the point and polygon plots.